pulsar-client-cpp/python/pulsar/schema/definition.py

```diff
@@ -60,8 +64,10 @@ class Record(with_metaclass(RecordMeta, object)):
     # This field is used to set namespace for Avro Record schema.
     _avro_namespace = None
 
-    def __init__(self, default=None, required_default=False, required=False, *args, **kwargs):
```

Comment:
Shall we need to keep the old configuration `required_default=False` for compatibility consideration?

Reply:
I put it back for compatibility